Solutions/Proofpoint On demand(POD) Email Security/Parsers/ProofpointPOD.yaml

```diff
@@ -314,4 +379,245 @@ FunctionQuery: |
         | project-away Guid1
         | project-rename Urls = set_URLLIST
     };
-    union isfuzzy=true ProofpointPOD_message_view, ProofpointPOD_maillog_view
\ No newline at end of file
+    let ProofpointPOD_messageV2_view = view () { 
+        ProofpointPODMessage_CL
+        | extend 
+                FilterModulesUrldefenseCountsNoRewriteIsLargeMsgPartSize= toreal(parse_json(filter).modules.urldefense.counts.noRewriteIsLargeMsgPartSize),
```

Comment:
very big column name? is this new or existing field?

Reply:
@vakohl , these are the existing columns which were parsed with old table/parser as well.